### DISCUSSION TITLE
Glitches before mixing tracks (short breaks on every mixing process)

### DISCUSSION THREAD

**Discussion by unudelaradio** [AzuraCast/AzuraCast#6809] [General]
One thing i've noticed since i use AzuraCast, not only on recent versions, is that on some point, after starting a station, after a while - days or weeks, some glitches appear on mixing process. Before a track ends, on its last seconds it stops for a moment, then the jingle starts to play. And so on with the jingle too. Every track stops for a moment before the mixing process.

This problem dissapers only if i restart the AutoDJ. Then after a few days or weeks appears again.

Hope that all the issues with these glitches and also the ones with the bad crackles after mixing some tracks will be solved soon.

All the best!

**Comment by BusterNeece** [comment]
This is a known issue and likely an upstream thing caused by Liquidsoap: check out #6736 for more information.
